### ISSUE TITLE
Does anything break if I start it up with `sudo`?

### ISSUE THREAD

**Issue by mikermcneil** [vulncheck-oss/cli#58]
Just a thought I wondered when reading the readme.  (A common problem i ran into a lot in a past life) 

Cool stuff yall!

**Comment by tcampbPPU** [comment]
No, nothing should break if you start it up with sudo. The script is designed to be run as a normal user, the only thing that might happen is that the files created by the command could be owned by the root user, which might cause some issues if you want to modify them later on. But other than that, there should be no problems with running the program with sudo.
